### PR TITLE
deno: add French translation

### PR DESCRIPTION
**Pull request by Ceres445**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/deno.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+- Exécuter un fichier avec l'accès au réseau activé:
+
+`deno run --allow-net {{chemin/du/fichier.ts}}`
+
+- Exécuter un fichier à partir d’une URL:
+
+`deno run {{https://deno.land/std/examples/welcome.ts}}`
+
+- Installer un script exécutable à partir d’une URL:
```
Comment (kbdharun) [inline pages.fr/common/deno.md:22]:
```suggestion
> Plus d'information : <https://deno.land>.

- Exécuter un fichier JavaScript ou TypeScript :

`deno run {{chemin/du/fichier.ts}}`

- Démarrer un REPL (shell interactif) :

`deno`

- Exécuter un fichier avec l'accès au réseau activé :

`deno run --allow-net {{chemin/du/fichier.ts}}`

- Exécuter un fichier à partir d’une URL :

`deno run {{https://deno.land/std/examples/welcome.ts}}`

- Installer un script exécutable à partir d’une URL :
```

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr and thanks for your contribution.

Not a native speaker so I can't check the contents. I went through the syntax alone. For French pages generally, we add a space between the contents and the colon.

**Review by kbdharun — APPROVED** [review]

File: pages.fr/common/deno.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# deno
+
+> Un environnement d’exécution sécurisé pour JavaScript et TypeScript.
+> Plus d'information : <https://deno.land>.
+
+- Exécuter un fichier JavaScript ou TypeScript :
```
Comment (noraj) [inline pages.fr/common/deno.md:6]: Command and example descriptions on pages in French must use the third person singular present indicative tense, cf. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#french-specific-rules

**Review by noraj — CHANGES_REQUESTED** [review]
Thanks for your contribution, one little remark.

**Pull request merged**